doc/user-guide/vtk_to_pyvista.rst

```diff
@@ -93,18 +93,18 @@ Here, PyVista has done several things for us:
 
 #. The name for the :attr:`point_array <pyvista.point_array>` is given
    directly in dictionary-style fashion. Also, since VTK stores data
-   on the heap (linear segments of RAM memory; a C++ concept), the
-   data must be flattened and put in FORTRAN ordering (controls the
-   data "endianness"; numpy uses "C"-style endianness by
```

Comment:
Endianness refers to the order of bytes within a multibyte variable (e.g. an int64). This is something different.

Reply:
Quite right.  Thanks for fixing this.